lib/applications.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const model = require('./applications/model');
+
+module.exports = {
+  model,
+};
```

Comment:
Can we start to follow the pattern in https://github.com/punchcard-cms/punchcard/issues/328 and have this be `lib/applications/index.js` instead of `lib/applications.js`?